frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -40,10 +40,18 @@ class OverrideForkPointDialog
 
   override fun createCenterPanel() =
       panel {
-        row(
-            getString(
-                "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name")) {
+        row {
```

Comment:
I must done it in that way, coz when I try to use format with `<html></html>` tags this dialog cuts the last word  :cry: